vnext/Microsoft.ReactNative/Views/FlyoutViewManager.cpp

```diff
@@ -425,33 +430,37 @@ winrt::FlyoutPresenter FlyoutShadowNode::GetFlyoutPresenter() const {
 
 FlyoutViewManager::FlyoutViewManager(const Mso::React::IReactContext &context) : Super(context) {}
 
-const char *FlyoutViewManager::GetName() const {
-  return "RCTFlyout";
+const wchar_t *FlyoutViewManager::GetName() const {
+  return L"RCTFlyout";
 }
 
 XamlView FlyoutViewManager::CreateViewCore(int64_t /*tag*/) {
   return winrt::make<winrt::react::uwp::implementation::ViewPanel>().as<XamlView>();
 }
 
-facebook::react::ShadowNode *FlyoutViewManager::createShadow() const {
+ShadowNode *FlyoutViewManager::createShadow() const {
   return new FlyoutShadowNode();
 }
 
-folly::dynamic FlyoutViewManager::GetNativeProps() const {
-  auto props = Super::GetNativeProps();
-
-  props.update(
-      folly::dynamic::object("horizontalOffset", "number")("isLightDismissEnabled", "boolean")("isOpen", "boolean")(
-          "placement", "number")("target", "number")("verticalOffset", "number")("isOverlayEnabled", "boolean"));
-
-  return props;
+void FlyoutViewManager::GetNativeProps(const winrt::Microsoft::ReactNative::IJSValueWriter &writer) const {
+  Super::GetNativeProps(writer);
+  React::WriteProperty(writer, L"horizontalOffset", L"number");
+  React::WriteProperty(writer, L"isLightDismissEnabled", L"boolean");
+  React::WriteProperty(writer, L"isOpen", L"boolean");
+  React::WriteProperty(writer, L"placement", L"number");
+  React::WriteProperty(writer, L"target", L"number");
+  React::WriteProperty(writer, L"verticalOffset", L"number");
+  React::WriteProperty(writer, L"isOverlayEnabled", L"boolean");
 }
 
-folly::dynamic FlyoutViewManager::GetExportedCustomDirectEventTypeConstants() const {
-  auto directEvents = Super::GetExportedCustomDirectEventTypeConstants();
-  directEvents["topDismiss"] = folly::dynamic::object("registrationName", "onDismiss");
+void FlyoutViewManager::GetExportedCustomDirectEventTypeConstants(
+    const winrt::Microsoft::ReactNative::IJSValueWriter &writer) const {
+  Super::GetExportedCustomDirectEventTypeConstants(writer);
 
-  return directEvents;
+  writer.WritePropertyName(L"topDismiss");
```

Comment:
General question: 
Should we adopt a style for writing nested objects to make it easier to read  by using cpp scopes to indicate js object nesting...
For example:
```cpp

writer.WritePropertyName(L"topDismiss");
writer.WriteObjectBegin();
{
    React::WriteProperty(writer, L"registrationName", L"onDismiss");
}
writer.WriteObjectEnd();

```